google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -406,6 +408,92 @@ public void testJsonStreamWriterWithDefaultStream()
     }
   }
 
+  @Test
+  public void testJsonStreamWriterSchemaUpdate()
```

Comment:
We need to add a concurrent append test.

Reply:
I think there should be test already written in that fashion but was removed earlier.

Reply:
Test case added